src/renderer/Components/Login/index.jsx

```diff
@@ -1,14 +1,132 @@
+import { useState, useEffect } from "react";
+import { useNavigate } from "react-router";
+import { Link } from "react-router-dom";
+import axios from "axios";
+
+import {
+  signInWithPopup,
+  GoogleAuthProvider,
+  signInWithEmailAndPassword,
+} from "firebase/auth";
+
+import { auth } from "../../firebase";
+import usePackageStore from "@renderer/store";
+
 import googleLogo from "../../assets/images/googleLogo.svg";
 import kakaoLogo from "../../assets/images/kakaoLogo.svg";
 
 function Login() {
-  const handleKakaoLogin = () => {
-    window.location.href = `https://kauth.kakao.com/oauth/authorize?client_id=${import.meta.env.VITE_KAKAO_KEY}&redirect_uri=${import.meta.env.VITE_BASE_URL}&response_type=code`;
+  const navigate = useNavigate();
+  const { setClientStatus } = usePackageStore();
+
+  const [emailValue, setEmailValue] = useState("");
+  const [passwordValue, setPasswordValue] = useState("");
+
+  useEffect(() => {
+    if (window.Kakao && !window.Kakao.isInitialized()) {
+      window.Kakao.init(import.meta.env.VITE_KAKAO_KEY);
+    }
+  }, []);
+
+  const handleLocalLogin = async (event) => {
```

Comment:
```suggestion
  const handleEmailLogin = async (event) => {
```
local 은 보편적이지 않은 표현이 강하니 email이 어떨까 싶습니다 :)

Reply:
좋은 제안이라고 생각됩니다!👍 바꾸도록 하겠습니다!